packages/component/src/Tooltip/private/Tooltip.tsx

```diff
@@ -0,0 +1,26 @@
+import cx from 'classnames';
+import React, { memo, type ReactNode } from 'react';
+import { useStyleSet } from '../../hooks';
+
+type TooltipProps = Readonly<{
+  children: ReactNode;
+  className?: string | undefined;
+  position?: 'block-start';
+}>;
+
+function Tooltip({ children, className, position = 'block-start' }: TooltipProps) {
```

Comment:
Instead of `children` which potentially allow other people to put HTML or rich text in the tooltip, should we use a `text: string` prop instead?

Reply:
They can use markup like bold tags etc. As this is internal, I think we are responsible for managing limitations.